buku.1

```diff
@@ -100,8 +101,12 @@ Bookmarks with immutable titles are listed with '(L)' after the title.
 .PP
 .IP 13. 4
 \fBAlternative DB file\fR:
-  - The option --db (to specify an alternative database file location) is app-only. Manual usage is prone to issues arising from human error.
+  - The option \fB--db\fR (to specify an alternative database file location) is app-only. Manual usage is prone to issues arising from human error.
   - Note that this option is useful if you want to store the db file in cloud synced location. Another mechanism could be to have the db file synced and create a symlink to it at the default location.
+  - When a the argument to \fB--db\fR contains neither `.` nor directory separator, it's considered a \fIname\fR and is resolved as the matching file with `.db` extension within the default DB directory.
```

Comment:
@jarun I seem to have missed a typo here ("a the") :sweat_smile:

You may want to fix that before the release.

Reply:
Done.